### PR TITLE
fix: totalGasLimit and totalUniqueGasLimit are correct

### PR DESCRIPTION
**Pull request by frrist**
- values were flipped

### CODE REVIEW

**Comment by iand** [comment]
Happy with the fix but we also need to correct existing data

**Comment by frrist** [comment]
@iand in 258c3f3503ce06a354da78369f02b5b414dac32c I have added a command that will attempt to recompute the incorrect gas values.

File: commands/fixgas.go [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+	if err := db.DB.RunInTransaction(ctx, func(tx *pg.Tx) error {
+		err := tx.ModelContext(ctx, &msgs).Select()
+		if err != nil {
+			return err
+		}
+		return nil
+	}); err != nil {
+		return err
+	}
+	for _, msg := range msgs {
```
Comment (iand) [inline commands/fixgas.go:73]: There are 383k rows in this table. Maybe the select will work but I don't know about the update. Is there pagination support in go-pg?

Comment (frrist) [reply commands/fixgas.go:73]: I'm not sure, looking now.

File: commands/fixgas.go [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+		msg.GasLimitTotal = total
+		msg.GasLimitUniqueTotal = unique
+		msg.BaseFee = baseFee
+		msg.BaseFeeChangeLog = math.Log(baseFeeChangeF) / math.Log(1.125)
+		msg.GasFillRatio = float64(total) / float64(len(ts.Blocks())*build.BlockGasTarget)
+		msg.GasCapacityRatio = float64(unique) / float64(len(ts.Blocks())*build.BlockGasTarget)
+		msg.GasWasteRatio = float64(total-unique) / float64(len(ts.Blocks())*build.BlockGasTarget)
+	}
+	batchSize := 1000
```
Comment (frrist) [inline commands/fixgas.go:103]: I've added batch here, if you'd like I can make this configurable via an env var.

**Comment by iand** [comment]
I haven't run this. I don't expect to merge this PR, will just run from branch

**Comment by frrist** [comment]
@iand I have moved the command that fixes the table values to: #258. I think we will want to merge this PR to address the bug but agree that command should not be merged.

**Comment by frrist** [comment]
@iand any objection to merging this?

**Comment by iand** [comment]
No objections

**Pull request merged**